execution-service/src/main/resources/execution-service-defaults.properties

```diff
@@ -69,3 +69,11 @@ rvf.drools.rule.repository=https://github.com/IHTSDO/snomed-drools-rules.git
 rvf.drools.rule.branch=
 rvf.generate.mysql.binary.archive=false
 rvf.jdbc.data.myisam.folder=/var/lib/mysql
+
+#IMS
+ims.url=
+ims.username=
```

Comment:
Please name this item `rvf.ims.username` and also `rvf.ims.password`. The `ims.url` is fine as that is common among applications.